languages/csharp/exercises/concept/red-vs-blue-darwin-style/RedVsBlueDarwinStyle.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable RCS1163, RCS1110, IDE0060
+
```

Comment:
Like Mike, I've also checked if I get any warnings when removing this line, and for me it also doesn't result in any difference. Could you remove it?